pyportify/app.py

```diff
@@ -156,9 +156,10 @@ def transfer_playlists(request, s, g, sp_playlist_uris):
 
         tasks = []
         for i, sp_track in enumerate(sp_playlist_tracks):
-            query = SpotifyQuery(i, sp_playlist_uri, sp_track, track_count)
-            future = search_gm_track(request, g, query)
-            tasks.append(future)
+            if sp_track['track'] is not None:
```

Comment:
I think we should filter sp_playlist_tracks on line 136 instead of here, that way track_count is still accurate (since that track doesn't even show up in your playlist.  

We might possibly even want to filter it in s.fetch_playlist_tracks()